bridge/test/accusations/invalid-transaction-consumption-accusations.test.ts

```diff
@@ -0,0 +1,266 @@
+import { assert, expect } from "chai";
+import { InvalidTxConsumptionAccusation } from "../../typechain-types";
+import { Fixture, getFixture } from "../setup";
+import {
+  addValidators,
+  getAccusationDataForNonExistentUTXOWithInvalidSigGroup,
+  getInvalidAccusationDataWithSpendingValidDeposit,
+  getValidAccusationDataForNonExistentUTXO,
+  getValidAccusationDataForNonExistentUTXOChainId2,
+  getValidAccusationDataForNonExistentUTXOWithInvalidHeight,
+} from "./accusations-test-helpers";
+
+describe("StakingPositionDescriptor: Tests StakingPositionDescriptor methods", async () => {
```

Comment:
not sure if this is related to staking or accusations

Reply:
whoops - copy pasted the boilerplate but forgot to change the names here! Fixed now for this and the other tests too